From: Dmitry Shachnev <mitya57@debian.org>
Date: Fri, 10 Aug 2018 17:00:17 +0300
Subject: Do not assert that webfonts directory exists

We exclude that directory when building the DFSG tarball, and replace
the fonts with symlinks to packaged versions.

Forwarded: Not-Needed
---
 mkdocs/tests/build_tests.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/mkdocs/tests/build_tests.py b/mkdocs/tests/build_tests.py
index d2dd3ca..14ce321 100644
--- a/mkdocs/tests/build_tests.py
+++ b/mkdocs/tests/build_tests.py
@@ -550,7 +550,6 @@ class BuildTests(PathAssertionMixin, unittest.TestCase):
         self.assertPathIsDir(site_dir, 'js')
         self.assertPathIsDir(site_dir, 'css')
         self.assertPathIsDir(site_dir, 'img')
-        self.assertPathIsDir(site_dir, 'webfonts')
         self.assertPathNotExists(site_dir, '__init__.py')
         self.assertPathNotExists(site_dir, '__init__.pyc')
         self.assertPathNotExists(site_dir, 'base.html')
